.github/workflows/windows.yml

```diff
@@ -41,9 +41,11 @@ jobs:
             otp-version: '24'
             rebar3-version: '3.15'
             os: 'windows-latest'
+          - gleam-version: '0.19.0-rc3'
```

Comment:
For this version, without the change, CI would fail.

[After my change](https://github.com/paulo-ferraz-oliveira/setup-beam-example/actions/runs/3672093082/jobs/6207947498#step:2:18): ✅ 
[Before my change](https://github.com/paulo-ferraz-oliveira/setup-beam-example/actions/runs/3672093082/jobs/6207947498#step:3:22): ❌ 

(workflow file: https://github.com/paulo-ferraz-oliveira/setup-beam-example/actions/runs/3672093082/workflow)